ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -437,7 +437,7 @@ private void transitionLayoutOnlyViewToNativeView(
 
   private static boolean isLayoutOnlyAndCollapsable(@Nullable ReactStylesDiffMap props) {
     if (props == null) {
-      return true;
+      return false;
```

Comment:
Let's go with collapsableChildren, this will allow to keep the same API and could even be useful in the future for other cases where we want an index based API like stickyHeaderIndices.